SeleniumTests/LoginTests.cs

```diff
@@ -49,84 +49,90 @@ public void CloseBrowser()
 
-            //** NOTE: The Assert class has static methods that will fail a test if the condition does not pass
+            // asserts it displayed correctly
             Assert.IsTrue(errorMessageIsDisplayed, "No 'Invalid Username or Password' message displayed");
 
-            //The error message should disappear
+            // waits for the disappear
             Thread.Sleep(5000);
+
+            // tries to grab the message
             errorMessageIsDisplayed = _driver.FindElement(By.Id("errorMessage")).Displayed;
+
+            // assers it was gone
             Assert.IsFalse(errorMessageIsDisplayed, "Error message is displayed but should have faded out");
 
-            
+
         }
 
         [TestMethod]
         public void AValidUserName_WithAnInvalidPassword()
         {
-            //** HINT: View the login page in the brower,  Valid username and password combinations have been provided right from the login page
-            //         Examine the test above and write the test where the username is valid, but the password is invalid
-            throw new NotImplementedException("Test not implemented");
-            
-            /*Assert:
-             * Error message displays and then fades out
-             */
+            do_login("admin", "bad_password");
+
+            System.Threading.Thread.Sleep(200);
+
+            var errorMessageIsDisplayed = _driver.FindElement(By.Id("errorMessage")).Displayed;
+
+            Assert.IsTrue(errorMessageIsDisplayed, "No 'Invalid Username or Password' message displayed");
+
+            Thread.Sleep(5000);
+
+            errorMessageIsDisplayed = _driver.FindElement(By.Id("errorMessage")).Displayed;
+            Assert.IsFalse(errorMessageIsDisplayed, "Error message is displayed but should have faded out");
         }
 
         [TestMethod]
         public void A_Physician_Logs_In_Successfully()
         {
-            throw new NotImplementedException("Test not implemented");
-           
-            /*Assert:
-             * The physician is taken to the physician page (iemosoft.com/selenium-test-physician)
-             * HINT: use _driver.Url
-             */
+            // does doc login
+            do_login("dr smith", "P@ssword");
+
+
+            // waits for page to load
+            Thread.Sleep(3000);
+
+            // grabs url
+            String doc_url = _driver.Url;
+
+            // compares expected url to actual and assters they are equil
+            Assert.AreEqual("http://iemosoft.com/selenium-test-physician/", doc_url);
         }
 
         [TestMethod]
         public void A_Nurse_Logs_In_Successfully()
         {
-            throw new NotImplementedException("Test not implemented");
+            // does nurse login
+            do_login("nurse jones", "P@ssword");
+
+            // waits for page to load
+            Thread.Sleep(3000);
+
+            // grabs url
+            String doc_url = _driver.Url;
```

Comment:
the string alias is common in C#.  Again a convention shops will enforce.  https://programmingwithmosh.com/csharp/difference-between-string-and-string-in-c/